bicopter_pybullet_controller:
  bicopter_pybullet_controller_node:
    ros__parameters:

      # Safety Limits
      servo_limit_min_angle: -0.7854  # radians
      servo_limit_max_angle: 0.7854
      motor_power_limit: 0.5  # [0,1], 1.0 means max power

      # topics
      actuator_commands_topic: "actuator_commands"
      imu_readings_topic: "/mpu6050/imu_readings"

      # bicopter model
      urdf_path: "/home/ebnerl/dev_ws/src/bicopter/bicopter_model/urdf/bicopter_urdf.urdf"

      # physics
      prop_c1: 16.0  # hover should be around half throttle: Thrust = c * speed^2
                    # speed = 0.5, Thrust (1 prop) = 9.81 * 0.8kg / 2 ~ 4N --> c = 16


